web/packages/extension/src/background.ts

```diff
@@ -77,10 +235,13 @@ function onMessage(
 }
 
 (async () => {
-    const { ruffleEnable } = await utils.getOptions();
+    const { ruffleEnable, swfTakeover } = await utils.getOptions();
     if (ruffleEnable) {
         await enable();
     }
+    if (!swfTakeover) {
```

Comment:
Is this really necessary here? (We only conditionally enabled from `enable`.

Reply:
Yeah, this shouldn't be needed anymore, conditionally enabling in `enable` was the latest commit, after I realized without it if you toggled off both the SWF takeover and Ruffle itself the SWF takeover would be enabled when you re-enabled Ruffle even if you didn't toggle it back on.